conf/machine/iot2050.conf

```diff
@@ -20,3 +20,7 @@ IMAGE_TYPE ?= "wic-img"
 WKS_FILE ?= "iot2050"
 
 IMAGE_INSTALL += "u-boot-script"
+
+#swupdate
+BOOTLOADER = "u-boot"
+U_BOOT = "u-boot-iot2050-basic"
```

Comment:
Unrelated to this commit, but if the variables only make sense to SWUpdate, they should be prefixed.

Reply:
Ok will look upstream into it.